src/main/webapp/index.html

```diff
@@ -23,9 +23,7 @@
       <option value="ASCENDING" selected
         >Increasing Alphabetical Order (Title)</option
       >
-      <option value="DESCENDING"
-        >Decreasing Alphabetical Order (Title)</option
-      >
```

Comment:
I already formatted this file with prettier so I don't think it is necessary to change the formatting.

Reply:
It did that automatically, didn't realize it touched your files

Reply:
Yeah you can choose what files to add in the PR. So if for some reason you see a file that is not committed that you didn't edit then just ignore it.